Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/DecryptFileViewModel.cs

```diff
@@ -75,19 +66,30 @@ public DecryptFileViewModel(IDesignServices services) : base(services)
         /// </summary>
         public DesignInArgument<bool> ContinueOnError { get; set; } = new DesignInArgument<bool>();
 
+        /// <summary>
+        /// The file that you want to encrypt.
+        /// </summary>
+        public DesignInArgument<IResource> InputFile { get; set; } = new DesignInArgument<IResource>();
+
+        /// <summary>
+        /// The file that you want to encrypt.
+        /// </summary>
+        public DesignOutArgument<CryptographyRemoteItem> DecryptedFile { get; set; } = new DesignOutArgument<CryptographyRemoteItem>();
```

Comment:
This should be `ILocalResource` since the file is now locally available AND there's nothing crypto-specific about it (it's just a local resource like any other from a consumer's perspective, no extra bells and whistles about it).